### DISCUSSION TITLE
Orca Slicer "rescue mode" - ability to slice with first X number of layers deliberately empty

### DISCUSSION THREAD

**Discussion by DougJoseph** [OrcaSlicer/OrcaSlicer#5249] [Ideas]
If you've ever had a nice big model print fail, especially real close to being finished, you've probably done what I have, which is to painstakingly edit a copy of the sliced gcode file to delete the first X layers, to run a "rescue" version of the file.

When I recently started using Orca Slicer, I noticed the preview mode for the sliced model has a Z slider with two knobs. I got an idea from that lower knob, for a "rescue" mode in which an upwards drag of the lower knob would cause those layers below it to not be rendered in the saved gcode file, i.e. only output the layers between the two knobs.

A "rescue" mode should also offer choices about homing axes, such as order of axes homed, and whether or not to exclude an axis, and choices about bed temp and nozzle temp, etc.

This kind of feature could make it very easy to produce a gcode file that is a "rescue" version of the original.

Thoughts?

**Comment by primitiveType** [comment]
Yes, please.
